### ISSUE TITLE
Sequence annotations are ignored

### ISSUE THREAD

**Issue by lionelvillard** [openwhisk/openwhisk-wskdeploy#246]
Manifest causing the error:
```
package:
  name: test-public 
  sequences:
    protected:
      actions: test-public/web,test-private/sensitive
      annotations:
         web-export:true
  ...
```

I'm getting this error
```
Got error creating action with error message: Sequence component does not exist. (code 185541) and error code: 144.
```

**Comment by paulcastro** [comment]
Seems like `test-private/sensitive` isn't defined?  It either has to be installed already or eventually you can pull in `test-private` as a dependency once PR #243 is merged.

Annotation code is in deployment.yaml code.  We should copy this into manifest code, then refactor deployment code so it properly merges any annotations already declared in the manifest.

**Comment by lionelvillard** [comment]
`test-private/sensitive` exists, but the code creates `test-public/test-private/sensitive`.

**Comment by paulcastro** [comment]
Ugh.  That's fixed in PR#243.

**Comment by lionelvillard** [comment]
Ah yes I see it. What about annotations?

**Comment by paulcastro** [comment]
Nope, annotations is not fixed in #243 👎

**Comment by lionelvillard** [comment]
Actually this fix is still valid as the second part of the condition is not needed (If there is no `/` then there is no prefix as well).
